modules/gapi/cmake/detect_onevpl.cmake

```diff
@@ -0,0 +1,27 @@
+if(WITH_ONEVPL)
+  find_package(VPL QUIET HINTS "${ONEVPLROOT}")
```

Comment:
Usually cmake-style locations involve `_DIR` (like oneVPL_DIR in your case). Is this ONEVPLROOT set if oneVPL installed?